src/ontology/uberon-edit.obo

```diff
@@ -223456,6 +223475,16 @@ relationship: develops_from UBERON:6000112 ! insect dorsal ectoderm
 relationship: part_of UBERON:0000922 ! embryo
 relationship: part_of UBERON:6005569 ! insect presumptive embryonic/larval tracheal system
 
+[Term]
+id: UBERON:6005043
+name: insect trachea
+def: "Cuticle-lined epithelial tube that forms part of the insect tracheal system." [FBbt:00005043]
+synonym: "trachea" BROAD SENSU [NCBITaxon:50557]
+xref: FBbt:00005043
```

Comment:
Same as above.

Reply:
On second thought, shouldn’t `insect trachea` be made a subclass of `open tracheal system trachea` (UBERON:0003127)?